src/installing-deis/configuring-object-storage.md

```diff
@@ -15,10 +16,42 @@ Additionally, Deis ships with a [Minio](http://minio.io) [component](https://git
 
 # Telling Deis What to Use
 
-The Deis components determine what object storage system to use via environment variables that you set up:
+The Deis components determine what object storage system to use via environment variables that you set up. The below list is the lookup order for all Deis components.
```

Comment:
yup, added sections in https://github.com/arschles/deis-docs-v2/commit/238a71f7ff112a18f77ee911e42f9c8b9ce6f3db